tooling/tracer/Cargo.toml

```diff
@@ -16,3 +16,4 @@ noir_debugger.workspace = true
 noirc_artifacts.workspace = true
 noirc_printable_type.workspace = true
 runtime_tracing.workspace = true
+tracing = "0.1.40"
```

Comment:
Is this the right way of doing this? I'm not sure. I tried something like the lines above, but didn't try very hard to get it to work.

Reviewer: please, share your opinion.

Reply:
Using `tracing.workspace = true` should work perfectly fine because `tracing = "0.1.40"` is added in the main workspace environment and we should rather inherit it from the workspace dependencies.

Reply:
Tried that, but it didn't work. Will try again.

Reply:
It worked, thanks.